src/main/java/com/nike/cerberus/service/S3StoreService.java

```diff
@@ -170,4 +170,10 @@ private String getFullPath(final String path) {
         logger.debug("The hash for {} is {}", path, objectMetadata.getETag().toString());
         return Optional.ofNullable(objectMetadata.getETag());
     }
+
+    @Override
+    public void copyFrom(String sourceIdentifier, String path) {
```

Comment:
I understand what `sourceIdentifier` is, but I had to think about it. Some javadoc on this method will make it super clear

Reply:
Might be even better to use the S3 specific names here since the parameters names do not need to match the interface.

e.g. copyFrom(String sourceBucketName, String sourceS3Key)